### PR TITLE
Update option for hiding pre-release

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

File: OneDrive/deploy-and-configure-on-macos.md [hunk 1]
```diff
@@ -511,6 +511,11 @@ The example for this setting in the .plist file is:
 <string>(UpdateRing)</string>
 ```
 
+> [!NOTE]
+> If you want to hide the option, "Get pre-release Microsoft internal updates to display", you will need to opt into the Deferred update ring.
+> Example 'default write com.microsoft.OneDrive Tier -string "Deferred"'
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:516]:
```suggestion
> For example, `default write com.microsoft.OneDrive Tier -string "Deferred"`.                                                                      
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 2]
```diff
@@ -511,6 +511,11 @@ The example for this setting in the .plist file is:
 <string>(UpdateRing)</string>
 ```
 
+> [!NOTE]
+> If you want to hide the option, "Get pre-release Microsoft internal updates to display", you will need to opt into the Deferred update ring.
+> Example 'default write com.microsoft.OneDrive Tier -string "Deferred"'                                                                      
+
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:517]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@mkbond007 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cartergreen** [comment]
Both enterprise and deferred are supported here.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**